rust/kcl-lib/src/std/assert.rs

```diff
@@ -24,132 +24,124 @@ async fn _assert(value: bool, message: &str, args: &Args) -> Result<(), KclError
 /// Check that the provided value is true, or raise a [KclError]
 /// with the provided description.
 pub async fn assert(_exec_state: &mut ExecState, args: Args) -> Result<KclValue, KclError> {
-    let (data, description): (bool, String) = args.get_data()?;
-    inner_assert(data, &description, &args).await?;
+    let actual = args.get_unlabeled_kw_arg("actual")?;
+    let gt = args.get_kw_arg_opt("isGreaterThan")?;
+    let lt = args.get_kw_arg_opt("isLessThan")?;
+    let gte = args.get_kw_arg_opt("isGreaterThanOrEqual")?;
+    let lte = args.get_kw_arg_opt("isLessThanOrEqual")?;
+    let eq = args.get_kw_arg_opt("isEqualTo")?;
+    let epsilon = args.get_kw_arg_opt("epsilon")?;
```

Comment:
The epsilon was changed to optional. I'm not sure I like this. But I can't really think of a better alternative. It is admittedly annoying to always have to specify it. But we don't have an integer type. Integer math is an illusion. Everything is floating point.

I think having it be optional will create more subtle errors. All our simple examples are just comparing to literals, but in real programs, it will be the result of math operations, and they won't be exact.

Is there a way users could opt-in to an exact integer comparison, rather than opt-out?

Reply:
What if instead we had a default epsilon, idk, 10^-9? And then users could specify their own to override the default?

Reply:
Yes, I think that's good. Would `epsilon = 0` be the way to do an exact equality?

Could define it over here next to the tolerance constant.

https://github.com/KittyCAD/modeling-app/blob/7a90d029e15bdf1856e35e9bf6ff58ef2f0a1d4b/rust/kcl-lib/src/std/mod.rs#L290

Reply:
Yeah, I like having a non-zero default tolerance. I don't think there is a perfect solution here, but since we are in FP world and we do have tolerances in the engine, I think it is probably closer to user expectations to have that here rather than insist on 'correctness'

Reply:
Agree. Changed to tolerance.